constants/battle_constants.asm

```diff
@@ -276,11 +276,24 @@ DEF TEAM_LUCKY_CHANT EQU %00000111
 DEF TEAM_TAILWIND    EQU %00111000
 DEF TEAM_WISH        EQU %11000000
 
+; wPlayerThroatChopEmbargoCount or wEnemyThroatChopEmbargoCount bit masks
+DEF EMBARGO_MASK     EQU %00001111
```

Comment:
everything else like this is named with a prefix and without the `_MASK`, maybe this could be as well?